Makefile

```diff
@@ -9,7 +9,7 @@ MODULE := $(shell head -1 go.mod | cut -d' ' -f2)
 
 ARTIFACTS := _out
 PKGS ?= ./...
-TALOS_RELEASE ?= v0.7.0
```

Comment:
this is side-effect of pulling in new go-procfs with `.Get()` fix - it also pulls in new default KSPP args which are not compatible with 0.7

I have an issue in our backlog to decouple default args from go-procfs